src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -182,27 +187,23 @@ class __attribute__((visibility("default"))) TvgLottieAnimation
     }
 
     // Saver methods
-    bool save2Gif(string data, string mimetype, int width, int height, int fps)
+    bool save(string mimetype)
     {
-        errorMsg = NoError;
+        if (mimetype == "gif") {
+            return save2Gif();
+        } else if (mimetype == "tvg") {
+            return save2Tvg();
+        }
```

Comment:
please add "png" as well.

Reply:
> Please consider the png exporting as well.

ignored as we decided to build it on viewer